templates/rfc_template.md

```diff
@@ -0,0 +1,65 @@
+# Title of My Design
+
+
+## [Optional] Subtitle of my design 
+
+**Authors:** [Nice Person](mailto:some.nice.person@chanzuckerberg.com)
+
+**Approvers:** [Friendly Tech Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly PM Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly Tech Person2](mailto:some.nice.person@chanzuckerberg.com)
+
+##tl;dr 
```

Comment:
Think you need a space after `##` or it doesn't apply the markup

Reply:
Oops. Done